tests/lib/boot.sh

```diff
@@ -11,12 +12,16 @@ bootenv() {
     if [ $# -eq 0 ]; then
         if command -v "$GRUB_EDITENV" >/dev/null; then
             "$GRUB_EDITENV" list
+        elif [ -s "$GRUBENV_FILE" ]; then
+            cat "$GRUBENV_FILE"
```

Comment:
Is grub-editenv unavailable in core18? @mvo5 do we need it?

Reply:
Unfortunately it is unavailable on core18 - we don't need it on core18, snapd has code to read/write grubenv. We could add something like "snap debug list-bootenv" or something. But I think its ok to just use `grep -v '^#' /boot/grub/grubenv ` - the format of the grubenv is pretty simple.